chapters/annotations.tex

```diff
@@ -1328,6 +1329,10 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 \lstinline!caption! is a caption for display in the file dialog.
 \lstinline!loadSelector! is used to select an existing file for reading, whereas \lstinline!saveSelector! is used to define a file for writing.
 
+The presence of \lstinline!directorySelector! specifying \fmtannotationindex{Selector} suggests the use of a file dialog to select an existing directory.
+The \lstinline!filter! may not be used.
```

Comment:
Maybe it wouldn't hurt to point out that it's OK to allow the dialog to be used for creating a directory on-the-fly?
```suggestion
The selected directory does not need to exist at the time of opening the dialog; it is allowed to let the dialog be used to create directory before selecting it.
The \lstinline!filter! may not be used.
```

Reply:
(Unfortunately I needed to attach this suggestion to the line below in order to keep conversations apart.)